### PR TITLE
Courtney

### PR DESCRIPTION
**Pull request by cwalker1332**

### CODE REVIEW

File: Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java [hunk 1]
```diff
@@ -12,28 +12,160 @@
...
+    public int getTs() {
+        return ts;
+    }
+
+    public void setTs(int ts) {
+        this.ts = ts;
+    }
+
+
+    public Attachment(String fallback, String color, String pretext, String author_name, String author_link, String author_icon, String title, String title_link, String text, String image_url, String thumb_url, String footer, String footer_icon, int ts) {
```
Comment (Fromssa-Olana) [inline Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java]: I do not think we need another constructor. We will be populating our field variables from the Json that is In-coming . you can delete it.

File: Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java [hunk 2]
```diff
@@ -12,28 +12,160 @@
...
+    // TODO: determine if we need fields
+    // Fields fields;
+
+    private String image_url;
+    private String thumb_url;
+    private String footer;
+    private String footer_icon;
+    private int ts;
 
     public Attachment(JSONObject json) {
```
Comment (Fromssa-Olana) [inline Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java:34]: We will be using this constructor instead. I will try the parsing part once you corrected the file.

**Review by Fromssa-Olana — DISMISSED** [review]

File: Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java (hunk 1 above)
Comment (cwalker1332) [reply Lab_SlackBot/src/com/floreo/bbah/model/Attachment.java]: Fixed it

**Review by Fromssa-Olana — APPROVED** [review]
Great work!

**Pull request merged**